### PR TITLE
hide back button in NavBar within admin panel

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Admin v1 shows the `BackButton` in the `NavBar` which isn't supposed to.

This PR adds `hasBackButton={false}` for the Admin v1 use cases.

image

### CODE REVIEW

File: src/components/molecules/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -101,6 +101,9 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
 
   const isOnPlaya = pathname.toLowerCase() === venueInsideUrl(PLAYA_VENUE_ID);
 
+  // @debt remove when back button is removed from NavBar and/or admin v1 is no longer used
+  const isInAdmin = useRouteMatch("/admin/");
```
Comment (goran-peoski-work) [inline src/components/molecules/NavBar/NavBar.tsx:105]: [NB] Not using a constant in `settings.ts` as to contain  the `@debt` for this particular case in this particular line/file and not spread it further.

**Review by mike-lvov — DISMISSED** [review]

**Review by 0xdevalias — CHANGES_REQUESTED** [review]
This is super hacky. `NavBar` already has a prop `hasBackButton` which defaults to true. In admin, that prop should be set to false.

File: src/components/molecules/NavBar/NavBar.tsx (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/molecules/NavBar/NavBar.tsx]: reverted change in bbcc9c48252441cff5298c00280c10ac2c3eef35

**Review by 0xdevalias — APPROVED** [review]
Haven't checked if this is all the places, but code diff looks good now. If you're happy that it covers all the things, then LGTM!

**Comment by mike-lvov** [comment]
The alternative for this PR that popped into my head is to take the BackButton from NavBar and only render it in TemplateWrapper

**Comment by goran-peoski-work** [comment]
> to take the BackButton from NavBar

This is already mentioned in `@debt` comment above the use of it, but felt like being more conservative ATM would be more prudent. Just addressing the bug in hand.

**Pull request merged**